eml/api/api.js

```diff
@@ -1,20 +1,34 @@
-import axios from 'axios';
+import axios from "axios";
 
-const testUrl = 'http://localhost:8888';
-const testExpo = 'http://172.30.243.206:8888'; //Change to local expo ip
-const digitalOcean = 'http://207.154.213.68:8888';
+const testUrl = "http://localhost:8888";
+const testExpo = "http://172.30.245.130:8888"; //Change to local expo ip
+const digitalOcean = "http://207.154.213.68:8888";
 
 const url = testUrl;
 
 /*** COURS, SECTIONS AND EXERCISES ***/
 
+export const getCourseByid = async (courseId) => {
+  const res = await axios.get(url + "/api/courses/" + courseId);
+  return res.data;
+};
+
+export const getSectionByid = async (sectionId) => {
+  const res = await axios.get(url + "/api/sections/" + sectionId);
+  return res.data;
+};
+
+export const getExerciseByid = async (exerciseId) => {
+  const res = await axios.get(url + "/api/exercises/" + exerciseId);
+  return res.data;
+};
+
 // Get specific course
 
 export const getCourse = async (courseId) => {
   try {
-    const res = await axios.get(url + '/api/courses/' + courseId)
+    const res = await axios.get(url + "/api/courses/" + courseId);
```

Comment:
This function is doing the same as the new getCourseByid function. The code should be refactored so it uses the same one everywhere, and the redundant one should be deleted/out-commented